### PR TITLE
added eventsdata

### PR DESCRIPTION
**Pull request by ShravaniAK**
added recent events data of gdsc iiitv.

### CODE REVIEW

**Comment by ShravaniAK** [comment]
resolve #7

File: src/Components/Events/util/EventsData.js [hunk 1]
```diff
@@ -43,6 +45,30 @@ const EventsData = [
...
+  },
+  {
+    thumbnail: 'https://i.imgur.com/GzHFcj2.jpg',
+    title: "Open Source Workshop",
+    date: "1/10/2022 6pm",
+    text: "The workshop was  exclusively for Second year students who aspire to participate in the Contribute-a-thon-22 as well as Hacktoberfest",
+  },
+  {
+    thumbnail: 'https://i.imgur.com/KJA3EzC.jpg',
+    title: "Building Apps with Flultter",
```
Comment (Im-Madhur-Gupta) [inline src/Components/Events/util/EventsData.js:68]: Correct the typo.

**Review by Im-Madhur-Gupta — CHANGES_REQUESTED** [review]
Please remove the top 4 events from the EventsData array in EvetsData.js.

File: src/Components/Events/util/EventsData.js (hunk 1 above)
Comment (ShravaniAK) [reply src/Components/Events/util/EventsData.js:68]: sir i am not getting , which typo needs to be corrected?

Comment (Im-Madhur-Gupta) [reply src/Components/Events/util/EventsData.js:68]: Check the spelling of Flutter.

**Review by Im-Madhur-Gupta — CHANGES_REQUESTED** [review]
Fix the typo, everything else looks good.

**Review by Im-Madhur-Gupta — APPROVED** [review]
LGTM

**Pull request merged**